src/elements/kw-share-shell/demo/index.html

```diff
@@ -0,0 +1,165 @@
+            @apply --layout-center;
+            @apply --layout-end-justified;
+            padding-top: 5%;
+        }
+        
+        .bottom .content .actions button img {
+            position: relative;
+            top: 2px;
+            left: 2px;
+        }
+        
+        .orangebutton {
+            background-color: #FF8413;
+            border-radius: 5px;
+            width: 79px;
+            height: 32px;
+            color: white;
+            border: none;
+        }
+        
+        .remixbutton {
+            width: 79px;
+        }
+        
+        .bottom .line {
+            height: 1px;
+            width: 100%;
+            display: block;
+            background-color: rgba(99, 107, 126, .24);
+        }
+        
+        div[tombstone].bottom .likes,
+        div[tombstone].bottom .comments,
+        div[tombstone].bottom .shares {
+            width: 80px;
+            height: 20px;
+            background: rgba(0, 0, 0, 0.01);
+            color: transparent;
+        }
+    </style>
+</head>
+
+<body>
+    <div class="vertical-section-container centered" style="background-color:#27323A">
+        <h3>Basic kw-share-shell demo</h3>
+        <demo-snippet>
+            <template>
+                <h3>with share data</h3>
+                <kw-share-shell id="detail">
+                    <div slot="share-slot">
+                    </div>
+                </kw-share-shell>
+                <script>
+                        document.addEventListener('WebComponentsReady', function () {
+                            var shell = document.querySelector('kw-share-shell#detail');
+                            var share = {
+                                title: 'My awesome share',
+                                description:"Please note that the new version does not use deep, and therefore requires you to import the dom-modules in every element that needs to use them.",
+                                user: {
+                                    username: 'platypusmeister',
+                                    avatar: {
+                                        urls: {
+                                            circle: 'https://s3-eu-west-1.amazonaws.com/world.kano.me/users/avatars/578bab1e7457798011f0ce63/avatar-circle.png'
+                                        }
+                                    }
+                                },
+                                cover_url: 'http://vignette2.wikia.nocookie.net/adventuretimewithfinnandjake/images/0/0b/Blush.jpg/revision/latest',
+                                date_created: new Date(),
+                                likes: new Array(27),
+                                comments_count: 5
+                            }
+                        
+                           shell.share = share;
+                        });
+                    </script>
+            </template>
+        </demo-snippet>
+    </div>
+</body>
+
+</html>
```

Comment:
The demo doesn't need to display a full share-card. It is supposed to show the functionality of the component alone, not in a specific case